sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.3.15 (2023-09-13)
+## 12.3.15 (2023-09-18)
 
 ### Bugs Fixed
 - Fixed bug where delete entity did not work on entities with empty primary keys.[(33390)](https://github.com/Azure/azure-sdk-for-java/issues/36690)
```

Comment:
Since you're updating the CHANGELOG, could you update this to mention the `API name` so that it's more discoverable?

Reply:
nit: other changelogs use the phrasing `Fixed the issue with...` rather than `Fixed bug`